pmda/util.py

```diff
@@ -177,3 +177,31 @@ def make_balanced_slices(n_frames, n_blocks, start=None, stop=None, step=None):
     slices[-1] = slice(last.start, last_stop, last.step)
 
     return slices
+
+def second_order_moments(S1, S2):
+    """
+    Given the partial centered moments of two partitions (S1 and S2) of a data
+    set S, calculates the second order moments of S = S1 ∪ S2.
+
+    Parameters
+    ----------
+    S1 : Array
+
+    S2 : Array
+
+    Returns
+    -------
+    S : Array
+
+    References
+    ----------
+    .. [CGL1979] T. F. Chan, G. H. Golub, and R. J. LeVeque. "Updating
+    formulae and a pairwise algorithm for computing sample variances."
+    Technical Report STAN-CS-79-773, Stanford University, Department of
+    Computer Science, 1979.
```

Comment:
Also cite the other paper from which you got the notation. Use the newer paper's notation for the Notes and algorithmic description.